### DISCUSSION TITLE
Wazuh Agent Groups

### DISCUSSION THREAD

**Discussion by Hammer-Bear** [Security-Onion-Solutions/securityonion#5723] [Unsupported Versions] [answered]
Hi, 

Trying to set up agent groups in Wazuh which we used to do in the standalone however the binary for creating the groups doesn't seem to exist or be where the documentation said it should be (https://documentation.wazuh.com/3.13/user-manual/reference/tools/agent_groups.html?) as we don't have the Wazuh interface is this still possible via terminal or has the function not been included in the SO version?

**Comment by matt00davis** [comment]
Did you get an update Dan-Bull? The agent_control script is also missing which was useful for listing connected agents etc but we've got a work around for this

**Comment by lan94** [comment, marked as answer]
How did you try to run `agent_groups`? Security Onion runs Wazuh Manager in a Docker container, which means you need to run your command in your container _so-wazuh_.
For example, if you want to list your existing groups, you could run the following:
`sudo docker exec -it so-wazuh /var/ossec/bin/agent_groups -l`

**Reply by matt00davis** [reply]
Thanks Ian, just been having a look:

# docker exec -it so-wazuh /var/ossec/bin/agent_groups -a -g windows
Do you want to create the group 'windows'? [y/N]: y
Group 'windows' created.
# docker exec -it so-wazuh /var/ossec/bin/agent_groups -a -i 106 -g windows
Do you want to add the group 'windows' to the agent '106'? [y/N]: y
Group 'windows' added to agent '106'.
# docker exec -it so-wazuh /var/ossec/bin/agent_groups -l -g windows
1 agent(s) in group 'windows':
  ID: 106  Name: xxxxx

**Reply by Hammer-Bear** [reply]
I was going off the wazuh document but also had a look in the location of where that script should be even under /opt/so/wazuh etc but couldn’t see it so that’s when I asked if it actually exists but if the above works then happy days
